sql/2021/cms/cms_adoption_2021_compared_to_2020.sql

```diff
@@ -0,0 +1,52 @@
+#standardSQL
+  # CMS adoptions, compared to 2020
+SELECT
+  _TABLE_SUFFIX AS client,
+  2021 AS year,
+  COUNT(0) AS freq,
+  total,
+  COUNT(0) / total AS pct
+FROM
+  `httparchive.technologies.2021_08_01_*`
```

Comment:
I think we're using 2021_07_01 this year in the Almanac, unless something changed

Reply:
Good catch, that's correct